tooling/did-method-registry-entry.yml

```
@@ -39,6 +39,10 @@ properties:
     description: https://www.w3.org/TR/did-core/#dfn-verifiable-data-registry
     type: string
     maxLength: 512
+  conformanceReport:
+    description: A URL to a DID Test Suite Conformance Report for the DID Method
```

Possibly this should be `An array of URLs that dereference to...` instead
```suggestion
    description: A URL that dereferences to a DID Test Suite Conformance Report for the DID Method
```